client/lcd/swagger-ui/swagger.yaml

```diff
@@ -23,6 +23,7 @@ tags:
   description: Query app version
 schemes:
 - https
+host: fabo.interblock.io:1317
```

Comment:
@jackzampolin @mircea-c this is weird.

If you copy this raw file into [https://editor.swagger.io/ ](https://editor.swagger.io/ ) and try to execute any `GET` request I get `TypeError: Failed to fetch`, but  if I use say `lcd.nylira.net` (without port) on this line the requests return OK with the right response. 

Is anything wrong with the fabo node ?

Reply:
cc: @greg-szabo

Reply:
```
Access to fetch at 'https://fabo.interblock.io:1317/stake/validators' from origin 'https://editor.swagger.io' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

Reply:
Filed: #3227